argoverse/utils/cuboid_interior.py

```diff
@@ -98,7 +98,7 @@ def filter_point_cloud_to_bbox_2D_vectorized(bbox: np.ndarray, pc_raw: np.ndarra
     P1 = np.array([bbox[0][0:2]])
     P2 = np.array([bbox[1][0:2]])
     P4 = np.array([bbox[2][0:2]])
-    P5 = np.array([])
```

Comment:
this can be removed completely i believe

Reply:
👍